src/client/components/forms/publisher.js

```diff
@@ -29,7 +30,18 @@ const LoadingSpinner = require('../loading-spinner');
 const PublisherData = require('./parts/publisher-data');
 const RevisionNote = require('./parts/revision-note');
 
+/**
+ * React component to define a form to create and edit
+ * a Publisher.
+ * @extends React.Component
+ */
+
 class PublisherForm extends React.Component {
+
+	/**
+	 * Binds class methods to their respective data.
```

Comment:
The JSDoc website says [here](http://usejsdoc.org/howto-es2015-classes.html) that `@constructor` doesn't need to be used with ES6 classes. Do you still want the tag included?